src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { changeQuary, changeStatus } from '../../features/filter';
+import { Status } from '../../types/Status';
+import { RootState } from '../../app/store';
 
 export const TodoFilter: React.FC = () => {
+  const { status, query } = useSelector((state: RootState) => state.filter);
+
+  const dispatch = useDispatch();
+
+  const handleSelectFilter = (statusPayload: Status) => {
+    dispatch(changeStatus(statusPayload));
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    handleSelectFilter(event.target.value as Status);
+  };
+
+  const handleQuery = (queryPayload: string) => {
+    dispatch(changeQuary(queryPayload));
+  };
```

Comment:
this function receive event by default, change it
```suggestion
  const handleQuery = (e: ChangeEvent<HTMLInputElement>) => {
    dispatch(changeQuary(e.target.value));
  };
```